From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Valerii Sysoiev <valsysoiev@gmail.com>
Date: Sun, 17 Aug 2025 14:18:22 -0600
Subject: [PATCH 43/90] Fix TypeScript error - add mockUserContext to window
 interface


diff --git a/web/e2e/test-utils/enterprise.ts b/web/e2e/test-utils/enterprise.ts
index 5ea8789ce86e709d8bf895ea31c2b78103d51779..683fc69d9559e70502fe4cbbcfbd5bdf681210b3 100644
--- a/web/e2e/test-utils/enterprise.ts
+++ b/web/e2e/test-utils/enterprise.ts
@@ -6,6 +6,7 @@ declare global {
   interface Window {
     mockAADClaims?: any;
     mockAuthMode?: string;
+    mockUserContext?: any;
   }
 }
 
